docs/monitoring/08-03-create-and-configure-grafana-dashboard.md

```diff
@@ -7,7 +7,7 @@ This tutorial shows how to create and configure a basic Grafana dashboard of a [
 
 ## Prerequisites
 
-This tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) tutorial that uses the `monitoring-custom-metrics` example. This example deploys the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+This tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) tutorial that uses the `monitoring-custom-metrics` example. This example deploys the `sample-metrics-8081` service, which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
```

Comment:
```suggestion
This tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) tutorial that uses the `monitoring-custom-metrics` example. This example deploys the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
```
I would leave the comma out here, as the second part of the sentence defines the first one (this service exposes the metric, it's important here :)